documentation/man/features/build.rst

```diff
@@ -9,6 +9,8 @@ SYNOPSIS
 | *kw* (*b* | *build*) [(-i | \--info)] [\--alert=(s | v | (sv | vs) | n)]
 | *kw* (*b* | *build*) [(-n | \--menu)] [\--alert=(s | v | (sv | vs) | n)]
 | *kw* (*b* | *build*) [(-d | \--doc)] [\--alert=(s | v | (sv | vs) | n)]
+| *kw* (*b* | *build*) [(-c | \--cpu-scaling)] <percentage> [\--alert=(s | v | (sv | vs) | n)]
```

Comment:
I think it should be:
`[(-c | \--cpu-scaling) <percentage>]`
@kwy95 ?

Reply:
@kwy95

Reply:
indeed you're right

Reply:
My bad, I missed this comment somehow.